### DISCUSSION TITLE
[PvP] Season start

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#6833] [Suggestions]
Given the influx of more players on season start into battlegrounds, I would suggest to heavily increase the required players for the battlegrounds to start again on a temporary basis, to be closer to the blizzlike experience.

2vs2 warsongs or 4vs4 arathis arent enjoyable at all, far away from what is blizzlike and how the battlegrounds are designed to work

**Comment by tbcxd** [comment]
I don't think we will run into this issue anymore, not until most active pvp-ers have their new pvp offparts/s1 gear/weapons.
We should wait a little and see how things go once the above mentioned items are added for honor
People won't play bgs just for fun or title, they'll play them for gear as well.
Increasing the requirements for bgs to pop will completly kill bgs during NA hours
